podcasts/TranscriptFilter.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+
+protocol TranscriptFilter {
```

Comment:
I renamed this to `TranscriptFilter` to avoid any confusions with the filters we have in the app.